### PR TITLE
[MIG][10.0]sale_product_set

### PR DESCRIPTION
**Pull request by leemannd**
Migration of sale_product_set to V10

### CODE REVIEW

**Comment by leemannd** [comment]
@Yajo @pedrobaeza If you are interested, this one is ready.

**Comment by pedrobaeza** [comment]
We are preferring this other approach, although not exactly the same purpose: https://github.com/OCA/product-attribute/pull/180

**Comment by leemannd** [comment]
@pedrobaeza I like the approach of the bundle. AFAIK our goal is just to quickly add set of products to a quotation/order. So it's fully covered by this one.

**Comment by pedrobaeza** [comment]
OK, no problem, they don't have the exact scope indeed.

**Review by simahawk — APPROVED** [review]

**Review by gurneyalex — APPROVED** [review]

File: sale_product_set/wizard/product_set_add.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-from openerp import models, fields, api, _
+from odoo import models, fields, api, _
 import openerp.addons.decimal_precision as dp
```
Comment (yvaucher) [inline sale_product_set/wizard/product_set_add.py:5]: Can you also fix the two field's strings here by removing _()

Plus the typo on s/_descritpion/_description

**Comment by leemannd** [comment]
@yvaucher Thanks for the comment. Updated.

**Review by yvaucher — APPROVED** [review]

**Pull request merged**